### PR TITLE
Update Expeditor config to 0.5.0

### PR DESCRIPTION
**Pull request by tduffield**
Signed-off-by: Tom Duffield <tom@chef.io>

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Documentation available at http://http://expeditor-docs.es.chef.io/
```
Comment (ryancragun) [inline .expeditor/config.yml]: double protocol in the url

Comment (tduffield) [reply .expeditor/config.yml]: ah, nice catch. thanks

**Comment by srenatus** [comment]
restarted one travis job. unrelated flaky test.

**Comment by stevendanna** [comment]
Waiting to merge until @tduffield gives the all clear.

**Comment by tduffield** [comment]
All clear!

**Pull request merged**